### PR TITLE
update the community information page

### PR DESCRIPTION
**Pull request by thesuperzapper**
This PR improves the "About" / "Community" page in many ways

### Notable changes

- Action buttons are now present to highlight the "join slack" and "join mailing list" links 
- There is now an aggregated calendar that displays all community calendars/meetings in one place
- Updates the "weekly community meeting" times to reflect the new `8:00am London` meeting time

### Screenshot

Community-Kubeflow

### CODE REVIEW

**Comment by thesuperzapper** [comment]
Annoyingly because I had to edit a file at the root of the repo, only root OWNERS can approve this PR.

@zijianjoy or @james-jwu could you do a quick review (this is a straightforward update)?

/assign @zijianjoy @james-jwu

File: content/en/docs/about/community.md [hunk 1]
```diff
@@ -30,40 +28,56 @@ The Kubeflow Slack workspace has many channels, here are a few examples:
...
+| Kubeflow Community Call - Notes | [Google Doc](http://bit.ly/kf-meeting-notes)
+| Kubeflow Community Call - Recordings | [YouTube Playlist](https://www.youtube.com/playlist?list=PLmzRWLV1CK_ypvsQu10SGRmhf2S7mbYL5)
+
+## Kubeflow Community Calendars
 
-More detail about the Kubeflow weekly community call:
+The following calendars are maintained by [Kubeflow Working Groups](#kubeflow-working-groups) and contain meetings that you may wish to attend.
 
-| Description | Link |
+| Calendar | Link |
```
Comment (james-jwu) [inline content/en/docs/about/community.md:56]: Can you add Pipelines to the list of Calendars? Thanks!

Comment (thesuperzapper) [reply content/en/docs/about/community.md:56]: @james-wu can the pipelines calendar actually is the "Kubeflow community" one.

Comment (thesuperzapper) [reply content/en/docs/about/community.md:56]: @james-jwu the pipelines calendar actually is the "Kubeflow community" one, so it's already included.

Possibly, I could include a row that says pipelines wg uses the community one, but that's probably not necessary?

Comment (james-jwu) [reply content/en/docs/about/community.md:56]: I see that Training is probably also under Kubeflow Community calendar. This is fine.

**Pull request merged**